### ISSUE TITLE
Peer review for hw01 is open

### ISSUE THREAD

**Issue by jennybc** [STAT545-UBC/Discussion#330]
I have made all the peer review assignments for hw01.

This should trigger two notifications (two emails, probably) to each of you, corresponding to two issues I opened and assigned to you.

Each of these issues lives in the STAT 545 repository of a fellow student. That is the work you should review.

The instructions should be pretty clear. Just like with the homework itself, half the reason we are doing this is just to practice the mechanics. So don't stress out if it feels unfamiliar or you're not sure you're doing it right. This is why we practice.

~10~ 8 of you have not yet accepted my invitation to join the STAT 545 Organization, which prevented me from assigning you peer review. I emailed you. You know who are. Except for the one person whose email also bounced. I will light the beacons to communicate with you 😁.

There are definitely a few empty repos. If you are lucky enough to be assigned those for peer review, please just say so in the comment and consider that review done. You will not be penalized. We will gradually weed these out.

**Comment by HeshamMahrous** [comment]
@jennybc  Thanks Professor Jenny, 

I wanted to suggest something it can be more convenient. If it is easy for you can you make a pull request instead of sending emails to assign peer reviews. It is just because emails pill  up over the week and can be easily lost, but the pull requests can always stay in the repo and it would be easier to access it.

Thanks

**Comment by jennybc** [comment]
I don't send email to assign peer review. I open and assign GitHub issues. Which probably causes an email to be sent. But the email is NOT the primary trigger. The issue is.

You can see your open, assigned issues from the assigned tab of your Issues Dashboard:

https://github.com/issues/assigned

A pull request workflow would not work for our style of peer review. Unless you'd like to lay out a specific workflow? Pull requests are based on forks, for starters. I just don't see how it would work.

**Comment by HeshamMahrous** [comment]
Oh no thats perfect ! I didn't know that it shows on my issues. I actually use bitbucket most of the time and we used pull requests for code reviews before we merged our repos to master. I guess these are totally two different things but thanks for letting me know.

**Comment by jennybc** [comment]
Yeah I don't see who would be forking or PRing or merging what into what here. I use PRs for code review all the time. But it's not a great fit for the peer review we are doing.
